internal/cache/volume_cache_test.go

```diff
@@ -29,6 +29,29 @@ func TestVolumeCache(t *testing.T) {
 }
 
 func testCache(t *testing.T, when spec.G, it spec.S) {
+	when("#FilterReservedNames", func() {
+		when("volume contains a reserved name", func() {
+			it("modifies the volume name", func() {
```

Comment:
Are we filtering even when the docker host is Linux?

Reply:
It does, but we'll take another look at doing it only for Windows, so we don't bust folks' caches

Reply:
@natalieparellano We looked into this and we can make it Windows-container specific, however it makes the code a bit more complex as we have to key off of the image/dockerhost OS in `NewVolumeCache()`. That requires a `context.Context` object to be passed in, and changes the return values of `NewVolumeCache()` to include an `error`. Is it worth it to y'all?

If not, the only downside, is a one-time rebuild cost for the small subset of pack users who happened to have one of the reserved words in their hashed volume name

Reply:
I don't feel strongly about this. @jromero ?

Reply:
> I don't feel strongly about this.

Same. The cost vs benefit doesn't seem worth it at this point in time.